### PR TITLE
Wind parameters from .metaversefile

### PR DESCRIPTION
**Pull request by skaljac**
I have change everything commented in previous PR.

### CODE REVIEW

File: rig.js [hunk 1]
```diff
@@ -59,6 +69,23 @@ const localRaycaster = new THREE.Raycaster(); */
   return geometry;
 })(); */
 const _makeRig = app => {
+
+
+    function initWindParameters(rig) {
+        var hairPhysics = rig.app.getComponent('hairPhysics');
```
Comment (avaer) [inline rig.js:75]: Hair physics is not the best name, since all avatars should have hair physics. This is about wind.

File: rig.js [hunk 2]
```diff
@@ -59,6 +69,23 @@ const localRaycaster = new THREE.Raycaster(); */
...
+    
+        if (!hairPhysics) {
+            return;
+        }
+
+        if (!hairPhysics.hasOwnProperty('windDirectionX')) hairPhysics.windDirectionX = 0.0;
```
Comment (avaer) [inline rig.js:81]: Shouldn't the wind direction be gotten from the component?

Comment (avaer) [reply rig.js:81]: Components have a `value` which you can get look at.

**Comment by skaljac** [comment]
I have created new PR with fresh copy of webaverse:

https://github.com/webaverse/app/pull/1893

That is the reason for closing this PR

All requests are addressed there.

**Pull request closed without merging**